torch/csrc/jit/passes/quantization.h

```diff
@@ -118,6 +118,14 @@ TORCH_API void FoldConvBatchNorm2d(const script::Module& module);
  */
 TORCH_API void FoldQuantizeCallIntoBuffer(script::Module& module, const std::string& method_name);
 
+/** \brief Insert pack and unpack function in all graphs
+ *   of module
```

Comment:
This comment should probably be expanded with more details - otherwise it just repeats the function name, which is not helpful.